search/base.py

```diff
@@ -4,23 +4,23 @@
 from unittest.mock import patch
 
 from django.test import (
-    override_settings,
     TestCase,
 )
 
-from search.indexing_api import recreate_index
+from search import indexing_api
+from search.indexing_api import recreate_index, delete_index
 
 
-class ESTestCase(TestCase):
+class ESTestCaseWithIndex(TestCase):
```

Comment:
I went down this road because renaming every single test class that uses `ESTestCase` is going to be a lot of work, unless I find some kind of magic unix command to do it...